### ISSUE TITLE
codespaces: All required CI checks should be runnable inside the devcontainer / GitHub Codespaces

### ISSUE THREAD

**Issue by themarwhal** [magma/magma#11337]
Required CI checks: https://github.com/magma/magma/wiki/Contributing-Code#continuous-integration-ci--continuous-deployment-cd

Goal
1. Get all required CI checks to run in GitHub codespaces
2. Once 1 is done, document in the Wiki that any new check should be runnable in GitHub codespaces

Checks to test
- [x] DCO Check (N/A)
- [x] Semantic PR (N/A)
- [x] merge freeze (N/A)
- [ ] orc8r-build
- [ ] cloud-test
- [ ] cloud_lint
- [ ] insync-checkin
- [ ] feg-build
- [ ] feg-precommit
- [ ] nms-build
- [ ] eslint
- [ ] nms-flow-test
- [ ] nms-yarn-test 
- [ ] nms-e2e-test
- [x] lte-test
- [x] mme_test
- [ ] session_manager_test
- [ ] orc8r-gateway-test
- [ ] cwag-precommit
- [x] Magma-OAI-Jenkins (N/A)
- [ ] cwf-operator-build
- [ ] cwf-operator-precommit
- [ ] Python Format Check
- [ ] lint-clang-format
- [ ] markdown-lint
- [ ] C/C++ unit tests with Bazel

**Comment by shahrahman-fb** [comment]
Closing this issue as part of a repository cleanup effort. This issue has had no activity for over 2 years.

If this issue is still relevant, please reopen it or file a new issue with updated information. We are actively maintaining the project again and want to focus on current, actionable issues.

See: #15833 (Phase 1 cleanup), #15828 (tracking issue)
